package.json

```diff
@@ -2,6 +2,10 @@
   "name": "monex",
   "version": "4.2.1-SNAPSHOT",
   "description": "Monitoring Application for eXist-db",
+  "engines": {
+    "node": ">=18.12.1 <19",
```

Comment:
I think we should increase the range on the upper bound to something more recent, if we look here https://endoflife.date/nodejs version 22 seems to be the best choice

Reply:
I m going to test it now, I wanted to confirm the version number first.

Reply:
I think we can do it in this PR.